python/api/cluster/serializers.py

```diff
@@ -132,10 +135,12 @@ class ClusterUpdateSerializer(EmptySerializer):
         max_length=80,
         validators=[
             ClusterUniqueValidator(queryset=Cluster.objects.all()),
-            RegexValidator(
-                regex=settings.REGEX_CLUSTER_NAME,
-                code="WRONG_NAME",
-                msg="Name `{value}` doesn't meets requirements",
+            StartMidEndValidator(
+                start=string.ascii_letters + string.digits,
+                mid=string.ascii_letters + string.digits + r"-. _",
+                end=string.ascii_letters + string.digits,
+                err_code="WRONG_NAME",
+                err_msg="Wrong cluster name. Errors: `{errors}`",
```

Comment:
We can pass first part of `msg` - `Wrong cluster name.` and second part well be created while calling validator class. To remove error message "templating"

Reply:
refactored